### ISSUE TITLE
`aws_acm_certificate_*` rules reject valid `certificate_authority_arn`

### ISSUE THREAD

**Issue by k911** [terraform-linters/tflint-ruleset-aws#453]
Private certificate authority arn can have prefix `acm-pca` (AWS ACM PCA)

When provided valid PCA arn in resource `aws_acm_certificate` tflint outputs following error

Log:
```
Error - "arn:aws:acm-pca:eu-west-1:xxx:certificate-authority/xxxxxxx-..." does not match valid pattern ^arn:[\w+=/,.@-]+:acm:[\w+=/,.@-]*:[0-9]+:[\w+=,.@-]+(/[\w+=,.@-]+)*$ (aws_acm_certificate_invalid_certificate_authority_arn)
```

Example code:

```hcl
resource "aws_acm_certificate" "example" {

  certificate_authority_arn = "arn:aws:acm-pca:us-east-1:0000000000:certificate-authority/xxxxxx-xxx-xxx-xxxx-xxxxxxxxx"
  domain_name               = "example.internal"

  lifecycle {
    create_before_destroy = true
  }
}
```

**Comment by bendrucker** [comment]
This is a regression:

https://github.com/terraform-linters/tflint-ruleset-aws/commit/3ff053701c738cc1f5719298677f43202c7e3773#diff-80b7da6268ee2776db834bce16d11e78eb4d54ada38306001d915b23b8ff395bR32

**Comment by bendrucker** [comment]
Seems like this is a bad mapping:

https://github.com/terraform-linters/tflint-ruleset-aws/blob/master/rules/models/mappings/acm.hcl

Perhaps it was at one point valid and then the API model was tightened up:

https://github.com/aws/aws-sdk-go/blob/d0700c118beded1f78fe93ddfcb2c3bb53e7448c/models/apis/acm/2015-12-08/api-2.json#L852

So the original issue misstates the problem:

https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/acm_certificate#certificate_authority_arn

`certificate_authority_arn` is _always_ `acm-pca`, never plain `acm`. If you want a certificate issued by Amazon's public ACM authority, you just omit that attribute. 

So the fix should be as simple as updating the mapping.
